packages/debugger/package.json

```
@@ -73,7 +73,8 @@
     "@lumino/widgets": "^1.30.0",
     "@vscode/debugprotocol": "^1.51.0",
     "codemirror": "~5.61.0",
-    "react": "^17.0.1"
+    "react": "^17.0.1",
+    "vscode-debugprotocol": "~1.51.0"
```

This probably slips in following the rebase. `vscode-debugprotocol` has been renamed `@vscode/debugprotocol` and [that change](https://github.com/jupyterlab/jupyterlab/pull/11812) is already on master.

```suggestion
    "react": "^17.0.1"
```